generator/app.go

```diff
@@ -457,6 +460,41 @@ func createGroupReadme(groups []Group, prefix string) error {
 	return nil
 }
 
+func createAnnualReportIssue(groups []Group, prefix string) error {
+	// figure out if the user wants to generate one group
+	var selectedGroupName *string
+	if envVal, ok := os.LookupEnv("WHAT"); ok {
+		selectedGroupName = &envVal
+	}
+
+	for _, group := range groups {
+		if !strings.HasPrefix(group.Dir, "sig-") {
```

Comment:
This isn't necessary for this PR, but we may want to add a TODO so that we expand the template and enable this to take more prefixes so we can do the same thing for WG/UG/foo in the future.